src/vehicle.cpp

```diff
@@ -3744,6 +3786,60 @@ bool vehicle::can_float() const
     return draft_m < hull_height;
 }
 
+// apologies for the imperial measurements, theyll get converted before used finally in the vehicle speed at the end of the function.
+// sources for the equations to calculate rotor lift thrust were only available in imperial, and the constants used are designed for that.
+// r= radius or d = diameter of rotor blades.
+// area A [ft^2] = Pi * r^2 -or- A [ft^2] = (Pi/4) * D^2
+// Power loading [hp/ft^2] = power( in hp ) / A
+// thrust loading [lb/hp]= 8.6859 * Power loading^(-0.3107)
+// Lift = Thrust loading * power >>>[lb] = [lb/hp] * [hp]
+
+double vehicle::lift_thrust_of_rotorcraft( const bool fuelled, const bool safe ) const
+{
+    int total_diameter = 0;
+    for( const auto rotor : rotors ) {
+        total_diameter += parts[ rotor ].info().rotor_diameter();
+    }
+    int total_engine_w = total_power_w( fuelled, safe );
+    // take off 15 % due to the imaginary tail rotor power.
+    double engine_power_in_hp = ( ( total_engine_w ) / 1000 ) * 1.34102;
+    int rotor_area_in_feet = ( M_PI / 4 ) * std::pow( total_diameter * 3.28084, 2 );
+    // lift_thrust in lbthrust
+    double lift_thrust = ( 8.8658 * std::pow( engine_power_in_hp / rotor_area_in_feet,
+                           -0.3107 ) ) * engine_power_in_hp;
+    add_msg( m_debug, "lift thrust of %s = %f", name, lift_thrust );
+    add_msg( m_debug, "rotor area of %s in feet : %d", name, rotor_area_in_feet );
+    add_msg( m_debug, "engine_power_in_hp of %s : %f: ", name, engine_power_in_hp );
+    // convert to newtons.
+    return lift_thrust * 0.2248089431;
+}
+
+bool vehicle::has_sufficient_rotorlift() const
+{
+    // comparison of newton to kg
+    return lift_thrust_of_rotorcraft( true ) > to_kilogram( total_mass() ) * 0.102;
```

Comment:
Shouldn't `* 0.102` be on the left side? 1kg force = 9.8 newton near sea level, if I'm not mistaken.

Reply:
Yup ! thanks